libbeat/kibana/testdata/extensive/fields.yml

```diff
@@ -1110,7 +1110,7 @@
           fields:
             - name: kernel.pct
               type: scaled_float
-              format: percentage
+              format: percent
```

Comment:
Not fully sure about this change. Acording to docs, this is `percentage` https://www.elastic.co/guide/en/kibana/master/field-formatters-numeric.html

Reply:
Yes, I got surprised by that. We were using both, `percent` and `percentage`, but if you look at kibana only the ones defined as `percent` are working.
In documentation and in the UI it says `Percentage`, but internally it has to be `percent`. I guess that this is not so bad because this feature is more intended to be used from the Kibana UI.

Reply:
Ok. I'm glad that you have it under control :smile: